### PR TITLE
Fix position of `view-markdown-source`

### LINKED ISSUE #4239: `view-markdown-source` is on the wrong place on the page
image

Test on https://github.com/sindresorhus/refined-github/blob/main/privacy-policy.md

### PR DESCRIPTION
**Pull request by cheap-glitch**
Fix #4239 

## Test URLs
* https://github.com/sindresorhus/refined-github/blob/main/privacy-policy.md
* https://github.com/yargs/yargs/blob/master/docs/api.md

## Screenshot
image

### CODE REVIEW

File: source/features/view-markdown-source.tsx [hunk 1]
```diff
@@ -98,7 +98,7 @@ async function init(): Promise<void> {
 	const fileButtons =
 		select('.repository-content .Box-header.flex-md-items-center .d-flex') ??
 		// Pre "Repository refresh" layout
-		select('.repository-content .Box-header .d-flex')!;
+		select('.repository-content readme-toc .Box-header .d-flex')!;
```
Comment (fregante) [inline source/features/view-markdown-source.tsx:101]: > Pre "Repository refresh" layout

I don’t think this is the right part of the code

Comment (cheap-glitch) [reply source/features/view-markdown-source.tsx:101]: That's what I thought too but the first selector doesn't match anything in the DOM, while the second does.

**Review by yakov116 — APPROVED** [review]

File: source/features/view-markdown-source.tsx (hunk 1 above)
Comment (fregante) [reply source/features/view-markdown-source.tsx]: Wow, how can we have the same identical issue a year later? If the pre/post refresh code is creating issues, lets just drop it in this case.

File: source/features/view-markdown-source.tsx [hunk 2]
```diff
@@ -96,7 +96,7 @@ async function init(): Promise<void> {
 	delegate(document, '.rgh-md-rendered:not(.selected)', 'click', showRendered);
 
 	const fileButtons =
-		select('.repository-content .Box-header.flex-md-items-center .d-flex') ??
+		select('.repository-content readme-toc .Box-header .d-flex') ??
 		// Pre "Repository refresh" layout
 		select('.repository-content .Box-header .d-flex')!;
 	fileButtons.prepend(
```
Comment (fregante) [inline source/features/view-markdown-source.tsx:102]:
```suggestion
select('.repository-content readme-toc .Box-header .d-flex')!.prepend(

**Comment by fregante** [comment]
Also can you post a screenshot of what this looks like? It's always useful to have demo screenshots to remember what our code is expected to look like months later.

**Review by fregante — APPROVED** [review]
Untested

File: source/features/view-markdown-source.tsx [hunk 3]
```diff
@@ -95,11 +95,7 @@ async function init(): Promise<void> {
 	delegate(document, '.rgh-md-source:not(.selected)', 'click', showSource);
 	delegate(document, '.rgh-md-rendered:not(.selected)', 'click', showRendered);
 
-	const fileButtons =
-		select('.repository-content .Box-header.flex-md-items-center .d-flex') ??
-		// Pre "Repository refresh" layout
-		select('.repository-content .Box-header .d-flex')!;
-	fileButtons.prepend(
+	select('.repository-content readme-toc .Box-header .d-flex')!.prepend(
```
Comment (fregante) [inline source/features/view-markdown-source.tsx:98]: This selector has 2 elements:

Screen Shot


Let's use the new suggested way to find items more reliably. Maybe 
```suggestion
	select('#raw-url')!.closest('.d-flex')!.prepend(
```

Comment (cheap-glitch) [reply source/features/view-markdown-source.tsx]: Successfully tested this change on Firefox.

**Review by yakov116 — APPROVED** [review]

**Comment by fregante** [comment]
🎉 
Screen Shot 5

**Pull request merged**